mocks/handlers/current-user/devUser.ts

```diff
@@ -0,0 +1,38 @@
+import type { definitions, NullableValues } from '../../types';
+
+export const devUser: NullableValues<definitions['userData']> = {
```

Comment:
You'll notice that every mock type is wrapped into a `NullableValues<…>`.
`NullableValues<T>`  transforms recursively an object type to accept null values:
```ts
type Test1 = NullableValues<{ hello: string, or: { world: string } }>
// is equivalent to
type Test2 = { hello: string | null, or: { world: string | null } }
```

I had to do that because in the current spec, the nullable values are only marked as nullable via comments, and comments cannot be interpreted by a TS types generator.
Because of that, the generated types disallow null values where it can be possible. The workaround was to mark all the properties as nullable, that way we can mock as we want.